eng/Version.Details.xml

```diff
@@ -397,6 +393,11 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>0f3f8e1930c28b67f29990126bc2e8527e959a2e</Sha>
     </Dependency>
+    <!-- This package was dead-ended, pinning it until we decide whether or not to replace it with something else -->
+    <Dependency Name="Microsoft.CSharp" Version="4.7.0" Pinned="true">
```

Comment:
Actually, can't we remove this and place the reference in the manual section of Versions.props now?

Reply:
Yeah, and that might be better since the package is dead-ended. Plus it'd avoid any potential weird DARC behavior related to CPD's. I'll do that.